src/Sulu/Component/Content/Tests/Functional/Mapper/ContentMapperTest.php

```diff
@@ -2141,9 +2138,7 @@ public function testCopyLanguageTree()
     public function testNodeAndAncestorsExcludedGhosts()
     {
         $data = $this->prepareBigTreeTestData();
-        $child = $data[1]->getChildren()[0]->getChildren()[2]->getChildren()[1];
-
-        $result = $this->mapper->loadNodeAndAncestors($child->getUuid(), 'en', 'sulu_io', true);
+        $result = $this->mapper->loadNodeAndAncestors($data[8]->getUuid(), 'en', 'sulu_io', true);
```

Comment:
because its more readable in my opinion and i would have to reload the documents because of the proxy generation in the hydrate event.